Makefile

```diff
@@ -93,22 +96,29 @@ sim2: | build deps wakunode2
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim sim2 $(NIM_PARAMS) waku.nims
 
-scripts2: | build deps wakunode2
+example2: | build deps
 	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim scripts2 $(NIM_PARAMS) waku.nims
+		$(ENV_SCRIPT) nim example2 $(NIM_PARAMS) waku.nims
 
-protocol2:
+test2: | build deps
 	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim protocol2 $(NIM_PARAMS) waku.nims
+		$(ENV_SCRIPT) nim test2 $(NIM_PARAMS) waku.nims
 
-test2:
+scripts2: | build deps wakunode2
 	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim test2 $(NIM_PARAMS) waku.nims
+		$(ENV_SCRIPT) nim scripts2 $(NIM_PARAMS) waku.nims
 
-chat2:
+chat2: | build deps
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim chat2 $(NIM_PARAMS) waku.nims
 
+protocol2: | build deps
```

Comment:
Does anybody still care about this target? Else I remove it...

Reply:
can remove